Proofreading the Transition Guide (#11543)
openvinotoolkit/openvino: docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

## The Installer Package Contains OpenVINO™ Runtime Only

Starting from OpenVINO 2022.1, development tools are distributed only via [PyPI](https://pypi.org/project/openvino-dev/), and are no longer included in the OpenVINO installer package. For a list of those components, refer to the [installation overview](../../../install_guides/installing-openvino-overview.md) guide. Benefits of this approach include:

* simplification of the user experience - in previous versions, installation and usage of OpenVINO Development Tools differed from one distribution type to another (the OpenVINO installer vs. PyPI), 
* ensuring that dependencies are handled properly via the PIP package manager, and support virtual environments of development tools.

The structure of the OpenVINO 2022.1 installer package has been organized as follows:

- The `runtime` folder includes headers, libraries and CMake interfaces.
- The `tools` folder, which contains: [the compile tool](../../../tools/compile_tool/README.md), [deployment manager](../../OV_Runtime_UG/deployment/deployment-manager-tool.md), and a set of `requirements.txt` files with links to the corresponding versions of the `openvino-dev` package.

> - The `python` folder, which contains the Python version for OpenVINO Runtime.

Author: I'd say it is better to revert them. The use of colon suggests a list or series of items, which in the last case would be a bit "out of ordinary."

$ pot -h
```

> It is not required to install any other dependencies. For more details on the installation steps, see the [Install OpenVINO Development Tools](../../install_guides/installing-model-dev-tools.md).

Author: Exactly. Still, I find 'Installation of other dependencies is not required.' as a plausible option here.

## Clearer Library Structure for Deployment

> OpenVINO 2022.1 introduced a reorganization of the libraries, to make deployment easier. In the previous versions, it was required to use several libraries to perform deployment steps. Now you can just use `openvino` or `openvino_c` based on your developing language,  with the necessary plugins to complete your task. For example, `openvino_intel_cpu_plugin` and `openvino_ir_frontend` plugins will enable to load OpenVINO IRs and perform inference on the CPU device (for more details, see the [Local distribution with OpenVINO](../deployment/local-distribution.md)).

Author: ```suggestion
OpenVINO 2022.1 introduced a reorganization of the libraries, to make deployment easier. In the previous versions, it was required to use several libraries to perform deployment steps. Now you can just use `openvino` or `openvino_c` based on your developing language,  with the necessary plugins to complete your task. For example, `openvino_intel_cpu_plugin` and `openvino_ir_frontend` plugins will enable loading OpenVINO IRs and performing inference on the CPU device (for more details, see the [Local distribution with OpenVINO](../deployment/local-distribution.md)).
```

Enable + object  + to (verb)

or

Enable + noun (verb+ing)

> Below are detailed comparisons on the library structure between OpenVINO 2022.1 and the previous versions:

Author: ```suggestion
Below are detailed comparisons of the library structure between OpenVINO 2022.1 and the previous versions:
```

> * In 2022.1 release a single core library with all the functionalities (`openvino` for C++ Runtime, `openvino_c` for Inference Engine API C interface) is used, instead of the previous core libraries which contained `inference_engine`, `ngraph`, `inference_engine_transformations` and `inference_engine_lp_transformations`.

Author: ```suggestion
* In 2022.1 release, a single core library with all the functionalities (`openvino` for C++ Runtime, `openvino_c` for Inference Engine API C interface) is used, instead of the previous core libraries which contained `inference_engine`, `ngraph`, `inference_engine_transformations` and `inference_engine_lp_transformations`.
```

> * In 2022.1 release, a single core library with all the functionalities (`openvino` for C++ Runtime, `openvino_c` for Inference Engine API C interface) is used, instead of the previous core libraries which contained `inference_engine`, `ngraph`, `inference_engine_transformations` and `inference_engine_lp_transformations`.

Reviewer: ```suggestion
* Starting with 2022.1 release, a single core library with all the functionalities (`openvino` for C++ Runtime, `openvino_c` for Inference Engine API C interface) is used, instead of the previous core libraries which contained `inference_engine`, `ngraph`, `inference_engine_transformations` and `inference_engine_lp_transformations`.
```
